raiden/raiden_service.py

```diff
@@ -998,14 +998,14 @@ def _initialize_whitelists(self, chain_state: ChainState):
 
         events_queues = views.get_all_messagequeues(chain_state)
 
-        light_client_transports = self.transport.light_client_transports
+        light_client_transports = self.transport.light_clients
 
         for event_queue in events_queues.values():
             for event in event_queue:
                 if isinstance(event, SendLockedTransfer):
                     transfer = event.transfer
                     if transfer.initiator == self.address:
-                        self.transport.hub_transport.whitelist(address=transfer.target)
+                        self.transport.full_node.whitelist(address=transfer.target)
                 if isinstance(event, SendLockedTransferLight):
                     transfer = event.signed_locked_transfer
                     for light_client_transport in light_client_transports:
```

Comment:
i think we can just iterate over self.transport.light_clients here

Reply:
done